Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -948,8 +948,8 @@ public void InjectCustomUpperLimitWillCauseSutToReturnNumbersInLimit()
             var result = sut.Create<DoublePropertyHolder<int, long>>();
             // Verify outcome
             Assert.True(
-                (result.Property1 >= lower && result.Property1 < upper) &&
-                (result.Property2 >= lower && result.Property2 < upper)
+                (result.Property1 >= lower && result.Property1 <= upper) &&
+                (result.Property2 >= lower && result.Property2 <= upper)

```

Comment:
The previous implementation had the upper limit inclusive, as far as I understand. If you gave `[1; 255; 300]` as the limits, you would draw the first 254 from [1;254], __the 255th would be 255__ and then the last 256-300 woud be taken from [256;300] - this is due to [line 193 in RandomNumericSequenceGenerator.cs](https://github.com/jryanmoseley/AutoFixture/blob/master/Src/AutoFixture/RandomNumericSequenceGenerator.cs#L193). The issue here is only about the 255th draw in the previous example.

I believe the test wasn't correct to start with (at least from what I understand from the code); otherwise the code didn't follow the specs (but there is no mention in that in the class' comments, maybe this should be added too).